images/installer-origin-release/Dockerfile.ci

```diff
@@ -1,7 +1,7 @@
 # This Dockerfile is used as a CI job to build smoke tests for OpenShift installer
-# FIXME: Move this image to an openshift repo?
+# FIXME: Move this image to docker.io/openshift repo?
```

Comment:
Don't we push to [both docker.io and quay.io][1]?  If so, I think we should leave this as it was.

[1]: https://github.com/openshift/installer/pull/106#issue-206113988